astropy/constants/tests/test_sciencestate.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+
+def test_version_match():
+    from astropy import physical_constants, astronomical_constants
+    pversion = physical_constants.get()
+    import astropy.constants as const
```

Comment:
If there is a reason this import have to happen after `.get()`, it should be documented, lest in the future, someone else moves it back to the top.

Reply:
No reason for it now. Earlier, I was experimenting with deleting `units` and `constants` from `sys.modules`, but that caused units tests to fail.